clients/cloud/ruby/README.md

```diff
@@ -0,0 +1,68 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the [ ZenDesk Ruby Client for Apache Kafka](https://github.com/zendesk/ruby-kafka).
+
+
+# Prerequisites
+
+* [Bundler](https://bundler.io/) installed on your machine. Install via `gem install bundler`
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+    ```bash
+    $ cat ~/.ccloud/ccloud.config
+    bootstrap.servers=<broker-1,broker-2,broker-3>
+    sasl.username=<api-key-id>
+    sasl.password=<secret-access-key>
+    ```
+* Install gems
+    ```bash
+    $ bundle install
+    Resolving dependencies...
+    Using bundler 1.17.1
+    Using digest-crc 0.4.1
+    Using ruby-kafka 0.7.4
+    Bundle complete! 1 Gemfile dependency, 3 gems now installed.
+    Use `bundle info [gemname]` to see where a bundled gem is installed.
+    ```
+
+# Example 1: Hello World!
+
+In this example, the producer writes Kafka data to a topic in Confluent Cloud. 
+Each record has a key representing a username (e.g. `alice`) and a value of a count, formatted as json (e.g. `{"count": 0}`).
+The consumer reads the same topic from Confluent Cloud and keeps a rolling sum of the counts as it processes each record.
+
+1. Run the producer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the topic name:
+    ```bash
+    $ ruby producer.rb -f ~/.ccloud/ccloud.config --topic test1
+    Created topic test1
+    Producing record: alice	{"count":0}
+    Producing record: alice	{"count":1}
+    Producing record: alice	{"count":2}
+    Producing record: alice	{"count":3}
+    Producing record: alice	{"count":4}
+    Producing record: alice	{"count":5}
+    Producing record: alice	{"count":6}
+    Producing record: alice	{"count":7}
+    Producing record: alice	{"count":8}
+    Producing record: alice	{"count":9}
+    Producing record: alice	{"count":10}
+    10 messages were successfully produced to topic test1!
```

Comment:
I count 11 messages ;)

Reply:
Good catch, this and the consumer example were outdated - I had fixed it in the code but not here hehe